pkg/eval/debugger_test.go

```diff
@@ -156,7 +160,23 @@ func TestREPL_DumpScope(t *testing.T) {
 
 	result := output.String()
 	require.Contains(t, result, "(dump scope)\n")
-	require.Contains(t, result, "a:  s:\"hello\"")
-	require.Contains(t, result, "b:  b:false")
-	require.Contains(t, result, "c:  null:<>")
+	for k, v := range *scope {
+		require.Contains(t, result, fmt.Sprintf("%s:  %s", k, unaryString(v).GetS()))
+	}
+}
+
+func TestREPL_CallView(t *testing.T) {
+	filename := "../../cmd/sysl/tests/transform1.sysl"
```

Comment:
I like the idea of having a const called syslroot or something so there aren't so many dots everywhere